File: dal-dao-gen/dal_ftp/63/java/Entity/AbacusParaGen.java
package com.ctrip.dal.test.test4;

import java.sql.Timestamp;

import com.ctrip.platform.dal.dao.DalPojo;

public class AbacusParaGen implements DalPojo {
	private Integer paraID;
	private Integer paraTypeID;
	private String paraName;
	private String paraValue;
	private String description;
	private Integer abacusWSID;
	private Timestamp datachangeLasttime;
	public Integer getParaID() {
		return paraID;
	}

	public void setParaID(Integer paraID) {
		this.paraID = paraID;
	}

	public Integer getParaTypeID() {
		return paraTypeID;
	}

	public void setParaTypeID(Integer paraTypeID) {
		this.paraTypeID = paraTypeID;
	}

	public String getParaName() {
		return paraName;
	}

	public void setParaName(String paraName) {
		this.paraName = paraName;
	}

	public String getParaValue() {
		return paraValue;
	}

	public void setParaValue(String paraValue) {
		this.paraValue = paraValue;
	}

	public String getDescription() {
		return description;
	}

	public void setDescription(String description) {
		this.description = description;
	}

	public Integer getAbacusWSID() {
		return abacusWSID;
	}

	public void setAbacusWSID(Integer abacusWSID) {
		this.abacusWSID = abacusWSID;
	}

	public Timestamp getDatachangeLasttime() {
		return datachangeLasttime;
	}

	public void setDatachangeLasttime(Timestamp datachangeLasttime) {
		this.datachangeLasttime = datachangeLasttime;
	}

}

File: dal-dao-gen/dal_ftp/63/cs/Entity/AbacusParaGen.cs
using System;
using Arch.Data.Orm;

namespace com.ctrip.dal.test.test4.Entity.DataModel
{
    /// <summary>
    /// AbacusPara
    /// </summary>
    [Serializable]
    [Table(Name = "AbacusPara")]
    public partial class AbacusParaGen
    {
        /// <summary>
        /// 获取或设置ParaID
        /// </summary>
        [Column(Name = "ParaID",Length=10),PK]
        public int ParaID { get; set; }
        /// <summary>
        /// 获取或设置ParaTypeID
        /// </summary>
        [Column(Name = "ParaTypeID",Length=5)]
        public short? ParaTypeID { get; set; }
        /// <summary>
        /// 获取或设置ParaName
        /// </summary>
        [Column(Name = "ParaName",Length=50)]
        public string ParaName { get; set; }
        /// <summary>
        /// 获取或设置ParaValue
        /// </summary>
        [Column(Name = "ParaValue",Length=1000)]
        public string ParaValue { get; set; }
        /// <summary>
        /// 获取或设置Description
        /// </summary>
        [Column(Name = "Description",Length=50)]
        public string Description { get; set; }
        /// <summary>
        /// 获取或设置AbacusWSID
        /// </summary>
        [Column(Name = "AbacusWSID",Length=5)]
        public short? AbacusWSID { get; set; }
        /// <summary>
        /// 获取或设置DataChangeLastTime
        /// </summary>
        [Column(Name = "DataChangeLastTime",Length=23)]
        public DateTime? DataChangeLastTime { get; set; }
    }
}